### PR TITLE
Add an env variable to set metallb E2E tests configuration values 

### PR DESCRIPTION
**Pull request by sabinaaledort**
Adding this for the run_e2e step which requires the cluster type
to run the relevant test suite.

Also set the metallb repo to pull and test the d/s repo.

### CODE REVIEW

**Comment by sabinaaledort** [comment]
the change seems to be working as the skip is now set as expected: `Skipping none\|IPV6`.
the CI is failing as we don't have the updated tests in the d/s metallb repo yet.
`No idea what '--ipv4-service-range' is!`
the following pr will add it:
openshift/metallb#13

**Comment by fedepaol** [comment]
> We should also set the metallb repo to pull and test the d/s repo.

I'd reword this to "we must set / we set"

**Comment by fedepaol** [comment]
One nit about the commit message, otherwise lgtm

**Comment by sabinaaledort** [comment]
@russellb Hey, can you merge this for us and override the failing CI?
This is done as part of our work to sync the d/s metallb repo.
The CI is failing here because we don't have the updated tests in the d/s metallb repo yet. The following pr will add it: openshift/metallb#13
That's a chicken egg situation, each PR depends on the other.

**Comment by russellb** [comment]
Force merging this based on @sabinaaledort, the jobs are still broken, but won't affect any other repo
